client/src/modules/stock/stock.routes.js

```diff
@@ -10,14 +10,14 @@ angular.module('bhima.routes')
         },
       })
 
-      .state('stockMovements', {
-        url         : '/stock/movements',
-        controller  : 'StockMovementsController as StockCtrl',
-        templateUrl : 'modules/stock/movements/registry.html',
-        params : {
-          filters : [],
-        },
-      })
+    // .state('stockMovements', {
+    //   url         : '/stock/movements',
+    //   controller  : 'StockMovementsController as StockCtrl',
+    //   templateUrl : 'modules/stock/movements/registry.html',
+    //   params : {
+    //     filters : [],
+    //   },
+    // })
 
       .state('stockInlineMovements', {
```

Comment:
Can you replace this `stockInlineMovements` with `stockMovements`?  We have a lot of links from other registries to the `stockMovements` (Inventory, Purchase Orders, etc) and it would be nice to ensure those still worked.